app/code/Magento/Paypal/Model/SmartButtonConfig.php

```diff
@@ -67,7 +67,8 @@ public function getConfig(string $page): array
             'locale' => $this->localeResolver->getLocale(),
             'allowedFunding' => $this->getAllowedFunding($page),
             'disallowedFunding' => $this->getDisallowedFunding(),
-            'styles' => $this->getButtonStyles($page)
+            'styles' => $this->getButtonStyles($page),
+            'IsVisibleOnProductPage'  => (int)$this->config->getValue('visible_on_product')
```

Comment:
Can the name of this key be changed to proper camel case: `isVisibleOnProductPage ` (starting from the lower case letter)?

Reply:
ok